slothy/core/core.py

```diff
@@ -3429,6 +3427,9 @@ def is_good_enough( cur, bound ):
 
         ok = self._model.cp_model.status in [cp_model.FEASIBLE, cp_model.OPTIMAL]
 
+        # - Export (optional)
```

Comment:
It would be cleaner if this was before the call to `solve`?

Reply:
that doesn't work. it errors if you try that

Reply:
Why?

Reply:
more details: we printing stats about the solving in the export_model function. We would have to split that up if we want to export the model before solving.

Reply:
not doing that now